procedures/katello/release.md.erb

```diff
@@ -65,11 +65,20 @@
 - [ ] Wait for Foreman <%= foreman_version %>.0 GA to be packaged and built before proceeding
 <% end -%>
 - [ ] Update release version similar to [here](https://github.com/theforeman/theforeman-rel-eng/commit/2029a9688da00d9c385c3438dd71b594ba5f728e)
-- [ ] Update `katello`, `katello-repos` and `rubygem-katello` 
+- [ ] Update `katello`, `katello-repos` and `rubygem-katello`
   - [ ] Use <%= rel_eng_script('bump_rpm_packaging') %>: `PROJECT=katello VERSION=<%= short_version %> ./bump_rpm_packaging`
 - [ ] Merge packaging PR once job is green
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
```

Comment:
Glad I recently fixed this: 6ee0c080f390c7d1357c2cc557b8daeebb6bf9fb is just a week old.